Documentation/02 Arguments, Options, and Flags.md

```diff
@@ -360,3 +359,35 @@ Verbose: true, files: ["file1.swift", "file2.swift"]
 % example --file file1.swift file2.swift --verbose
 Verbose: false, files: ["file1.swift", "file2.swift", "--verbose"]
 ```
+
+## Alternative positional argument parsing strategies
+
+The default strategy for parsing arrays of positional arguments is to ignore  all dash-prefixed command-line inputs. For example, this command accepts a `--verbose` flag and a list of file names as positional arguments:
+
+```swift
+struct Example: ParsableCommand {
+    @Flag() var verbose: Bool
+    @Argument() var files: [String]
+    
+    func run() throws {
+        print("Verbose: \(verbose), files: \(files)")
+    }
+}
+```
+
+The `files` argument array uses the default `.remaining` parsing strategy, so it only picks up values that don't have a prefix:
+
+```
+% example --verbose file1.swift file2.swift
+Verbose: true, files: ["file1.swift", "file2.swift"]
+% example --verbose file1.swift file2.swift --other
+Error: Unexpected argument '--other'
+Usage: example [--verbose] [<files> ...]
+```
+
+The `.unconditionalRemaining` parsing strategy uses whatever input is left after parsing known options and flags, even if that input is dash-prefixed. If `files` were defined as `@Argument(parsing: .unconditionalRemaining) var files: [String]`, then the resulting array would also include strings that look like options:
+
```

Comment:
Minor nitpick, but I’d show the example here where `--verbose` is towards the end, and how it ends up being one of `files`, to make developers aware of that trade-off.

Reply:
`--verbose` gets parsed as a flag no matter where it's placed. I'll add a note that it makes the most sense to use `.unconditionalRemaining` when it's on the only property in the command.